src/funcs.c

```diff
@@ -1909,12 +1909,12 @@ void            ExecEnd (
 *F  FuncSetRecursionTrapInterval( <self>, <interval> )
 **
 */
-
+#include <stdio.h>
```

Comment:
Any reason why this #include is down here, and not at the top of the file with the others? For that matter: Why did you add it at all?